libsimprints/src/main/AndroidManifest.xml

```diff
@@ -1,12 +1 @@
-<manifest package="com.simprints.libsimprints"
-          xmlns:android="http://schemas.android.com/apk/res/android">
-
-    <application
-        android:allowBackup="true"
-        android:label="@string/app_name"
-        android:supportsRtl="true"
-        >
-
-    </application>
```

Comment:
As far as I know, if `application` is present this can't be imported as a library into another app.